docs/rules/sort-collections.md

```diff
@@ -10,10 +10,12 @@ Whenever npm changes package dependencies through `npm install`, it lexically so
 However, developers will manually update `package.json` and accidentally leave dependencies out of order.
 Doing so leads to "noise" in commits when a later change re-sorts the packages.
 
-Furthermore, the `"scripts"` collections in `package.json` files are generally easiest to work with when in alphabetical order.
-To keep related scripts close together in the file, consider using the common npm script convention of colon-separated namespaces, such as `watch:all` or `watch:dist`.
+For most collections (`dependencies`, `peerDependencies`, `exports` entries, etc.) this rule enforces a simple ascending lexical ordering (based on raw string keys).
 
-This rule aims to keep the dependency collections sorted in every commit.
+`scripts` are an exception: they use a lifecycle‑aware ordering, which groups `pre<name>` / `<name>` / `post<name>` together (even if the middle one is missing).
+Details & examples: see [Scripts Lifecycle Ordering](#scripts-lifecycle-ordering).
```

Comment:
[Docs] Just nitpicking, this is the only sentence not written like typical conversational flow.

```suggestion
See [Scripts Lifecycle Ordering](#scripts-lifecycle-ordering) for more details and examples.
```